src/StreamStats.jl

```diff
@@ -1,15 +1,15 @@
 module StreamStats
     import StatsBase, Distributions
+    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
+    import Base.maximum, Base.minimum
 
-    export update!, state, nobs, rand, ci
+    export update!, state, nobs, rand, ci, sample
```

Comment:
No need to export `Base.rand` (you don't export similarly extended `Base.show`)? I don't know how to think about whether independent `ci` functions should be exported both here and in [HypothesisTests.jl](https://github.com/JuliaStats/HypothesisTests.jl) -- thoughts?